setup.py

```diff
@@ -35,6 +35,7 @@ def read(fname):
           'Programming Language :: Python :: 2.7',
           'Programming Language :: Python :: 3',
           'Programming Language :: Python :: 3.2',
-          'Programming Language :: Python :: 3.3'
+          'Programming Language :: Python :: 3.3',
+          'Development Status :: 7 - Inactive'
       ],
       )
```

Comment:
I think it might be beneficial to print the warning in the setup.py as well as the __init__.py.  This way people will have a chance to see it upon install of the package as well as upon import of the package.